delocate/libsana.py

```diff
@@ -24,6 +31,9 @@ def tree_libs(start_path, filt_func=None):
         If None, inspect all files for library dependencies. If callable,
         accepts filename as argument, returns True if we should inspect the
         file, False otherwise.
+    ignore_missing : bool
```

Comment:
Do we want to ignore or warn?

Reply:
Currently `tree_libs` prints out ignored dependencies.  I don't want it to be silent since it seems very important when they get ignored.  Maybe 'skip_missing' would be a better term?